.circleci/config.yml

```diff
@@ -100,6 +100,18 @@ jobs:
     steps:
       - rust-tests:
           rust-version: "beta"
+  C tests:
+    docker:
+      - image: circleci/rust:latest
+    # We have to use a machine with more RAM for tests so we don't run out of memory.
```

Comment:
Do we?

Reply:
Sorry -- copy-and-pasted from the Rust tests.  I doubt the C test is memory intensive...

Reply:
Sorry -- copy-and-pasted from the Rust tests.  I doubt the C test is memory intensive...